### PR TITLE
Unstop a contact when we get an explicit user interaction such as follow

### PR DESCRIPTION
**Pull request by norkans7**
Fix https://sentry.io/nyaruka/rapidpro/issues/240135754/
and https://sentry.io/nyaruka/rapidpro/issues/240278120/

### CODE REVIEW

File: temba/triggers/models.py [hunk 1]
```diff
@@ -305,6 +311,10 @@ def find_and_handle(cls, msg):
             trigger.trigger_count += 1
             trigger.save()
 
+        if contact.is_stopped:
```
Comment (nicpottier) [inline temba/triggers/models.py]: Feels to me like we should have one or both of these, but not both.

Comment (norkans7) [inline temba/triggers/models.py]: find_and_handle is called for messages created by the server, the other one handle the msg inserted by Mage

Comment (nicpottier) [inline temba/triggers/models.py]: The other one is `catch_triggers` which just calls into this no?

Comment (norkans7) [inline temba/triggers/models.py]: It does not https://github.com/nyaruka/rapidpro/pull/1180/files#diff-229434cf35ddf9e28b7e51870fb6940aL259

Comment (nicpottier) [inline temba/triggers/models.py]: Hrmm.. ok, lets add a `ensure_unstopped` to contact that does this stuff then and is called before passing to the flow start.

**Pull request merged**